src/context/ArchiveContext/ArchiveContext.jsx

```diff
@@ -0,0 +1,71 @@
+import { useContext, createContext, useState, useEffect } from 'react';
+import { useAxios } from '../../CustomHooks';
+
+const ArchiveContext = createContext();
+
+const ArchiveProvider = ({ children }) => {
+	const [archiveNotes, setArchiveNotes] = useState([]);
+	const { response: archiveResponse, operation: fetchArchive } = useAxios();
+
+	useEffect(() => {
+		if (archiveResponse != null) {
+			if (archiveResponse.archives) {
+				setArchiveNotes(archiveResponse.archives);
+			}
+		} else {
+			getArchive();
+		}
+	}, [archiveResponse]);
+	const getArchive = () => {
+		fetchArchive({
+			method: 'GET',
+			url: '/api/archives',
+			headers: {
+				authorization:
+					'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI0OGJmMzFiYi1iYWQyLTQ3N2EtYmRhNy1kZDc4NDIxNDZhODgiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.kH6xXhMMvQ0sgsCz7e8vrSh1pVddplgC3IMzUSlTFsQ',
```

Comment:
Please dont use the hardcoded token over here.

Reply:
done